docs/src/tutorials/software-management.md

```diff
@@ -0,0 +1,49 @@
+# Software Management with thin-edge.io
+
+With thin-edge.io you can ease the burden of managing packages on your device.
+Software Management operates end to end from a cloud down to the OS of your device and reports statuses accordingly.
+
+## Software management components
+
+Software Management uses following components to perform software operations:
+
+### Cloud Mapper
+
+TBD
+
+### Tedge Agent
+
+TBD
+
+### Software Manager Plugin
+
+TBD
```

Comment:
This needs to be move under the architecture section. A tutorial needs no to focus on the aim of the feature and the basics to use it.